debian/prerm

```diff
@@ -25,6 +25,7 @@ case $1 in
 	# so we can't reliably restore the system back to that prior state.
 	#
 
+	systemctl reload delphix-platform.service
```

Comment:
Can you comment on this?

Reply:
Sure.. When a package is uninstalled, it'll attempt to remove all files/directories that that package owned; in this case, it'll attempt to remove `/var/lib/delphix-platform`.
                                                                                
If the package manager detects a file in that directory that isn't owned by the package, it won't remove the directory (presumably, in fear of removing some unrelated file). In this case, it's possible for the `/var/lib/delphix-platform/ansible-done` file to exist, which isn't owned by the package (even though this package's service will create the file). So, to remedy this, and ensure the `/var/lib/delphix-platform` directory is removed on uninstall, we should ensure that `ansible-done` file is removed first.
                                                                                
This is an existing issue, it's just not practically a problem since we generally don't remove the `delphix-platform` package. I just noticed it when doing some manual testing of this PR.
                                                                                
Another solution would be to store the `ansible-done` file in some other directory, and leave it there if it exists on uninstall; but I think it's better to just remove it.

Reply:
Sure.. When a package is uninstalled, it'll attempt to remove all files/directories that that package owned; in this case, it'll attempt to remove `/var/lib/delphix-platform`.
                                                                                
If the package manager detects a file in that directory that isn't owned by the package, it won't remove the directory (presumably, in fear of removing some unrelated file). In this case, it's possible for the `/var/lib/delphix-platform/ansible-done` file to exist, which isn't owned by the package (even though this package's service will create the file). So, to remedy this, and ensure the `/var/lib/delphix-platform` directory is removed on uninstall, we should ensure that `ansible-done` file is removed first.
                                                                                
This is an existing issue, it's just not practically a problem since we generally don't remove the `delphix-platform` package. I just noticed it when doing some manual testing of this PR.
                                                                                
Another solution would be to store the `ansible-done` file in some other directory, and leave it there if it exists on uninstall; but I think it's better to just remove it.

Reply:
I see, so the real purpose of this is to remove the `ansible-done` file.
Do you think we should just do that explicitly here, instead of relying on `systemctl reload`, since the link between the two isn't obvious when reading `man systemctl`:
```
       reload PATTERN...
           Asks all units listed on the command line to reload their configuration. Note that this will reload the service-specific configuration, not the unit configuration file of systemd. If you want systemd
           to reload the configuration file of a unit, use the daemon-reload command. In other words: for the example case of Apache, this will reload Apache's httpd.conf in the web server, not the
           apache.service systemd unit file.
```

Reply:
Yea, we could do that.

I thought that might leak the abstraction, but since it's all still contained in the same package, I think it's reasonable just to use `rm -f` here. That'll be more reliable if/when removing the package from a `chroot`, too.

Reply:
I made this change in my latest revision.